### DISCUSSION TITLE
项目能否部署于云服务器上

### DISCUSSION THREAD

**Discussion by XKlsy** [InternLM/MindSearch#74] [Q&A] [answered]
使用云服务器进行部署时，部分接口返回的127.0.0.1:3001进行查询，但实际应该时云服务器的ip才对，应该怎么调整代码或者怎么做

**Comment by Harold-lkk** [comment, marked as answer]
启动app的时候默认 ip 是0.0.0.0 是一个广播ip， 外部访问的时候访问云服务对应的ip即可(保证云服务ip外部可访问)
